templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} driver</h1>
```

Comment:
The `object|yesno:"Update,Create"` filter is used to determine whether the form is for updating or creating a driver. Ensure that the `object` context variable is correctly set to differentiate between these actions.